### PR TITLE
[DSY-1898] - Progress indicator update

### PR DESCRIPTION
**Pull request by gsleonel**
# Description

Updates the progress indicator component, inserting size and layer parameters.

Screenshot_1614022834

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit Test (DividerTest.kt)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: doc/progress-indicator.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Progress Indicator
 
 ### What is it?
-A user interface element that indicates the progress of an operation.
+The progress indicator is a component that informs users about the status of ongoing processes, such as loading an application, submitting a form, or saving updates.
```
Comment (marcosvbras) [inline doc/progress-indicator.md:4]: such as loading **_an_** application. Nessa parte em destaque não deveria ser "and"?

File: designsystem/src/test/kotlin/com/natura/android/progressindicator/ProgressIndicatorFixture.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    private var size: Int? = null,
+    private var layer: Boolean? = null,
+
+    private var context: Context = ApplicationProvider.getApplicationContext()
+) {
+    companion object {
+        private const val STANDARD = 0
+        private const val SEMI = 1
+        private const val MEDIUM = 2
+        private const val LARGE = 3
```
Comment (marcosvbras) [inline designsystem/src/test/kotlin/com/natura/android/progressindicator/ProgressIndicatorFixture.kt:18]: Pela característica linear dos calores das constantes, faria sentido talvez usar um Enum?

File: designsystem/src/main/kotlin/com/natura/android/progressindicator/ProgressIndicator.kt [hunk 3]
```diff
@@ -1,10 +1,112 @@
...
+    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
+        setMeasuredDimension(resources.getDimension(widthResourceAttribute).toInt(), resources.getDimension(heightResourceAttribute).toInt())
+    }
+
+    companion object {
+        const val STANDARD = 0
+        const val SEMI = 1
+        const val MEDIUM = 2
+        const val LARGE = 3
```
Comment (marcosvbras) [inline designsystem/src/main/kotlin/com/natura/android/progressindicator/ProgressIndicator.kt:110]: Pela característica linear dos valores das constantes, faria sentido talvez usar um Enum?

File: doc/progress-indicator.md (hunk 1 above)
Comment (gsleonel) [reply doc/progress-indicator.md:4]: nesse caso o an está sendo usado como _uma_: "como carregar uma aplicação.."

**Review by marcosvbras — APPROVED** [review]

**Pull request merged**